feature-ui-exam-result/src/main/java/team/duckie/app/android/feature/ui/exam/result/screen/quiz/QuizResultContent.kt

```diff
@@ -68,12 +73,29 @@ internal fun QuizResultContent(
             )
         }
         Spacer(space = 28.dp)
-        QuackAnnotatedText(
-            text = stringResource(id = R.string.rank_by_tag, mainTag, rank.toString()),
-            highlightTextPairs = persistentListOf(
-                mainTag to null,
-                rank.toString() to null,
-            ),
+        QuackText(
+            modifier = Modifier.align(Alignment.End),
+            annotatedText = buildAnnotatedString {
+                append("당신은 ")
+                withStyle(
+                    SpanStyle(
+                        color = QuackColor.DuckieOrange.composeColor,
+                        fontWeight = FontWeight.Bold,
+                    ),
+                ) {
+                    append(mainTag)
+                }
```

Comment:
여기 관련해서요. 좀 과할수도 있는데 요런 확장 함수 하나 만들어두면 편할 것 같아요.
```
@JvmInline
value class DuckieTextStyle(
    val spanStyle: SpanStyle,
) {
    companion object {
        val OrangeBold = DuckieTextStyle(
            SpanStyle(
                color = QuackColor.DuckieOrange.composeColor,
                fontWeight = FontWeight.Bold,
            )
        )
    }
}
```
```
inline fun <R : Any> AnnotatedString.Builder.withDuckieStyle(
    style: DuckieTextStyle,
    block: AnnotatedString.Builder.() -> R
): R {
    val index = pushStyle(style.spanStyle)

    return try {
        block(this)
    } finally {
        pop(index)
    }
}

```

그럼

```suggestion
            withDuckieStyle(DuckieTextStyle.OrangeBold) {
                append("안뇽")
            }
```

요렇게 바꿀 수 있게...

Reply:
꽥괙에 QuackTextStyle이 존재하고,아직 재사용 여지가 불분명하다고 생각되어서 private으로 일단 추상화 해놓았습니다.

[823ee66](https://github.com/duckie-team/duckie-android/pull/486/commits/823ee663ef53440027503de4c18f526f0792be62)